tests/functional/api/test_query.py

```diff
@@ -0,0 +1,14 @@
+from ape import accounts, chain
+
+
+def test_basic_query(test_provider):
+    a = accounts.test_accounts[0]
+    a.transfer(a, 100)
+    a.transfer(a, 100)
+    a.transfer(a, 100)
+    df = chain.blocks.query("number")
+    assert len(df) == 3
+    assert df["number"][0] == 0
```

Comment:
Do something more like this (but fill in the `...`)
```suggestion
    assert chain.blocks.query("number") == pd.Dataframe(...)
```